charts/tenant-base/chart/templates/deployment.yaml

```
@@ -51,13 +51,19 @@ spec:
             {{- toYaml .resources | nindent 12 }}
           volumeMounts:
             {{- toYaml .volumeMounts | nindent 12 }}
-          {{- with .secrets }}
           envFrom:
+          {{- with .secrets }}
             {{- range . }}
             - secretRef:
                 name: {{ printf "%s-%s" (include "chart.fullname" $) $name }}
             {{- end }}
           {{- end }}
```

If I understand Helm templates correct, the following is equivalent and more concise:
```suggestion
          {{- range .secrets }}
          - secretRef:
              name: {{ printf "%s-%s" (include "chart.fullname" $) $name }}
          {{- end }}
```